packages/wallets/src/evm/connectors/smart-wallet/types.ts

```diff
@@ -18,7 +18,7 @@ import { WalletConnectReceiverConfig } from "../../../core/types/walletConnect";
 export type SmartWalletConfig = {
   chain: ChainOrRpcUrl;
   factoryAddress: string;
-  thirdwebApiKey: string;
+  thirdwebApiKey?: string;
```

Comment:
Let's make this mandatory again?

Reply:
actually let's leave it optional, makes more sense when you're using your own bundler to not pass it